tactical-microgrid-standard/common/ControllerSelector.cpp

```diff
@@ -14,6 +18,9 @@ void ControllerSelector::got_heartbeat(const tms::Heartbeat& hb)
   Guard g(lock_);
   auto it = all_controllers_.find(hb.deviceId());
   if (it != all_controllers_.end()) {
+    ACE_DEBUG((LM_DEBUG, "(%P|%t) ControllerSelector::got_heartbeat: from mc \"%C\"\n",
+      hb.deviceId().c_str()));
```

Comment:
Could this and similar log messages be conditional, since they could produce a lot of logs in a live run? An option is a `--verbose` command-line option from the concrete devices that can be passed down here to use.

Reply:
Yes, I can. I committed them because I had to keep using them to figure out what was alive at a glance both locally and in CI.